Auth/ForgeTrust.AppSurface.Auth.Aspire.Keycloak/README.md

```
@@ -44,6 +48,110 @@ keycloak.Configuration.ApplyTo(web)
+```
+
+The fresh output directory contains a source snapshot under `themes/<name>` without development-only assets, a `Containerfile`, and an `appsurface-keycloak-theme-manifest.json` evidence file containing both source and packaged manifests. `Digest` binds the registration, both manifests, platform, base image, and template baseline. The Containerfile labels the theme name, source and packaged manifests, build-contract digest, base image, and platform. `VerifyPackagedTheme` fails if content is missing, added, or changed. The package does not build or push the image, mutate a production realm, deploy infrastructure, or automate rollout/rollback; those remain application and operations responsibilities.
+
+For a mutable source, recreate the build contract immediately before packaging. A source change after registration makes build-context materialization fail before it copies an unproven file into the image context.
+
+## Release Evidence
+
+After an application-owned CI build produces its digest-pinned theme image, bind it to the validated source and
+packaged manifests before any realm selection:
+
+    var releaseEvidence = AppSurfaceKeycloakThemeReleaseEvidence.Create(
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)